.github/workflows/coverage.yml

```diff
@@ -0,0 +1,84 @@
+name: Python coverage
+
+on:
+ push:
+   branches:
+     - main
+     - 'version-**'
+   tags: "*"
+ pull_request:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+env:
+  ERT_SHOW_BACKTRACE: 1
+  NO_PROJECT_RES: 1
```

Comment:
Dont think this is needed?

Reply:
The `tests/test_egg_simulation.py::test_run_egg_model` will run and fail without this env var.